### ISSUE TITLE
Vinli dummy start --dummy <dummyId> --vin <vin #> results in error

### ISSUE THREAD

**Issue by gordysc** [vinli/vinli-cli#5]
If I specify a VIN # to use when starting a route for a dummy device, I get the following error:

> Cannot read property 'length' of undefined

I'm executing the following command:

`vinli dummy start --dummy e30e9fe1-6740-40ea-82a8-16941e8976f6 --vin VVRHYR2TC2GV1BLPY`

And if I look at the vehicles listed on the device, the VIN appears to be correct:

**GET** https://platform.vin.li/api/v1/devices/e30e9fe1-6740-40ea-82a8-16941e8976f6/vehicles

```
{
  "vehicles": [
    {
      "id": "d1e8af07-a7bf-4b76-b6ca-ddf2a21b0f7e",
      "vin": "VVRHYR2TC2GV1BLPY",
      "make": null,
      "model": null,
      "year": null,
      "trim": null,
      "data": null,
      "createdAt": "2016-02-18T16:51:44.013Z",
      "lastStartup": "2016-02-18T16:51:42.568Z",
      "links": {
        "self": "https://platform.vin.li/api/v1/vehicles/d1e8af07-a7bf-4b76-b6ca-ddf2a21b0f7e",
        "trips": "https://trips.vin.li/api/v1/vehicles/d1e8af07-a7bf-4b76-b6ca-ddf2a21b0f7e/trips",
        "codes": "https://diagnostic.vin.li/api/v1/vehicles/d1e8af07-a7bf-4b76-b6ca-ddf2a21b0f7e/codes",
        "collisions": "https://safety.vin.li/api/v1/vehicles/d1e8af07-a7bf-4b76-b6ca-ddf2a21b0f7e/collisions"
      }
    },
    {
      "id": "e68cc4af-2d6f-4ba5-bb3c-3729d31f17b9",
      "vin": "VVU0GBLJHUADC619T",
      "make": null,
      "model": null,
      "year": null,
      "trim": null,
      "data": null,
      "createdAt": "2016-02-18T16:48:31.483Z",
      "lastStartup": "2016-02-18T16:48:29.525Z",
      "links": {
        "self": "https://platform.vin.li/api/v1/vehicles/e68cc4af-2d6f-4ba5-bb3c-3729d31f17b9",
        "trips": "https://trips.vin.li/api/v1/vehicles/e68cc4af-2d6f-4ba5-bb3c-3729d31f17b9/trips",
        "codes": "https://diagnostic.vin.li/api/v1/vehicles/e68cc4af-2d6f-4ba5-bb3c-3729d31f17b9/codes",
        "collisions": "https://safety.vin.li/api/v1/vehicles/e68cc4af-2d6f-4ba5-bb3c-3729d31f17b9/collisions"
      }
    }
  ],
  "meta": {
    "pagination": {
      "total": 2,
      "limit": 20,
      "offset": 0,
      "links": {
        "first": "https://platform.vin.li/api/v1/devices/e30e9fe1-6740-40ea-82a8-16941e8976f6/vehicles?offset=0&limit=20",
        "last": "https://platform.vin.li/api/v1/devices/e30e9fe1-6740-40ea-82a8-16941e8976f6/vehicles?offset=0&limit=20"
      }
    }
  }
}
```

**Comment by gordysc** [comment]
I addressed this with pull request #6

**Comment by gangstead** [comment]
I think the root problem was starting a route with a device id given as the dummy id should have thrown an error, but didn't.  In the patch release that just went out this should throw an error instead of picking an arbitrary dummy.

Also I found the source of the 'length of undefined' problem and fixed it.
